docs/overview.adoc

```diff
@@ -708,8 +708,9 @@ CALL apoc.periodic.rock_n_roll('match (p:Person) return id(p) as id_p', 'MATCH (
 
 [cols="1m,5"]
 |===
-| call apoc.refactor.cloneNodes([node1,node2,...],[withRelationships=false],[skipProperties=[]]) |  clone nodes with their labels and properties
-| call apoc.refactor.cloneNodesWithRelationships([node1,node2,...]) | deprecated
+| call apoc.refactor.cloneNodes([node1,node2,...],[withRelationships=false],[skipProperties=[]]) YIELD input, output, error |  clone nodes with their labels and properties
+| call apoc.refactor.cloneNodesWithRelationships([node1,node2,...]) YIELD input, output, error | deprecated
+| call apoc.refactor.cloneSubgraph([node1,node2,...], [rel1,rel2,...], [reanchorNodePairs=[]], [skipProperties=[]]) YIELD input, output, error | clone nodes with their labels and properties (optionally skipping specific properties), and clone the given relationships (will exist between cloned nodes only or be reanchored according to reanchor node pairings)
```

Comment:
should it also support a list of paths? that constitute the same subgraph?
Can we move config into a config map, so we're more flexible going forward?

Reply:
A list of paths may be a bit tricky, we would need to extract and de-duplicate the nodes and relationships from all paths. Easy enough to do in code with Sets, and I can see use cases for this. The problem is this messes a bit with the signature, 2 parameters for nodes and rels vs 1 parameter for paths. Remind me, can we overload procedure methods with the same name but different signatures? I didn't think we could. Maybe I could write a companion proc for this?

Reply:
I guess we can just have a single List<Object> parameter, which can be paths, relationships or a mix of both.
And we just use a function to extract the distinct nodes and rels from those.

Reply:
I'm just going to make a separate proc for it: apoc.refactor.cloneSubgraphFromPaths()